packages/eslint-plugin/tests/rules/prefer-optional-chain.test.ts

```diff
@@ -211,6 +211,7 @@ ruleTester.run('prefer-optional-chain', rule, {
     '!foo!.bar || !foo!.bar.baz;',
     '!foo!.bar!.baz || !foo!.bar!.baz!.paz;',
     '!foo.bar!.baz || !foo.bar!.baz!.paz;',
+    '!import.meta.url || true;',
```

Comment:
Maybe add `!new.target || true;` as well, it seems to work but it's exotic/meta enough to make me anxious

Reply:
+1, can't hurt to add the test!

Reply:
I can just add one in 🙂